integration/e2e/util.go

```diff
@@ -53,11 +53,22 @@ func BuildArgs(flags map[string]string) []string {
 	return args
 }
 
-func GetRequest(url string) (*http.Response, error) {
+func GetRequest(url string, h http.Header) (*http.Response, error) {
```

Comment:
Is the input header used? I can't find where we use it.

Reply:
Yeah this is a leftover, I needed this at some point to get credentials. But I 've found another way around. I'll revert that back.